packages/formdescriptioneditors/frontend/sirius-components-formdescriptioneditors/src/SplitButtonWidget.tsx

```diff
@@ -360,10 +363,10 @@ export const SplitButtonWidget = ({ widget, editingContextId, representationId }
       <div
         data-testid={`${widget.__typename}-Widgets-DropArea-${widget.id}`}
         className={classes.bottomDropArea}
-        onDragEnter={handleDragEnter}
-        onDragOver={handleDragOver}
-        onDragLeave={handleDragLeave}
-        onDrop={handleDrop}>
+        onDragEnter={readOnly ? noop : handleDragEnter}
```

Comment:
May be it would have been better/more readable to modify handleDragEnter/handleDragOver/handleDragLeave... instead.
What do you think about it?